src/main/java/de/tum/in/www1/artemis/domain/exam/Exam.java

```diff
@@ -461,7 +461,7 @@ public void setExamArchivePath(String examArchivePath) {
      */
     public enum ExamSearchColumn {
 
-        ID("id"), TITLE("title"), COURSE_TITLE("course.title"), EXAM_MODE("exam.testExam");
```

Comment:
Why do we not support sorting based on the course title?

Reply:
Hmm… it works for exercises and lectures but not for exams?

Reply:
@ge65cer I now fixed it by removing the "DISTINCT" instead in 810237674bb7a0cf3e3100a075e78e2e3ff410c0. Now sorting by course title also works :)